cdec/installation-manager-resources/src/main/resources/im-scripts/install-codenvy.sh

```diff
@@ -928,6 +941,15 @@ doCheckAvailableResourcesLocally() {
         exit 1;
     fi
 
+    if [[ ${writeAccessIssueFound} == true ]]; then
+        if [[ -n $INSTALL_DIRECTORY ]]; then
+            println "$(printError "ERROR: Installation directory") $(printImportantLink $INSTALL_DIRECTORY) $(printError "cannot be written to.")"
+        else
+            println $(printError "ERROR: Current directory doesn't have write access.")
```

Comment:
I don't think it is good message so as it doesn't say which user doesn't have access to which directory.